sentry/src/main/java/io/sentry/Breadcrumb.java

```diff
@@ -522,22 +528,21 @@ public Breadcrumb(@Nullable String message) {
     this.message = message;
   }
 
-  /**
-   * Returns the Breadcrumb's timestamp
-   *
-   * @return the timestamp
-   */
-  public long getTimestampMs() {
-    return timestamp;
-  }
-
   /**
    * Returns the Breadcrumb's timestamp as java.util.Date
    *
    * @return the timestamp
    */
+  @SuppressWarnings("JavaUtilDate")
   public @NotNull Date getTimestamp() {
-    return DateUtils.getDateTime(timestamp);
+    if (timestamp != null) {
```

Comment:
is it still showing improvements?
This looks like moving the issue from breadcrumb creation to event capture (as we sort the breadcrumbs). 
Or am i missing something?

Reply:
Yeah we only sort them when they are not created by us (=someone set them to the event), but I doubt anyone does that, so this will cover most of the cases probably: https://github.com/getsentry/sentry-java/blob/7e57220ee4ffa7af590fff675708884c4a07370c/sentry/src/main/java/io/sentry/SentryClient.java#L1054-L1058